webui/react/src/components/TaskList.tsx

```diff
@@ -116,8 +115,6 @@ const TaskList: React.FC<Props> = ({ workspace }: Props) => {
   const [tasks, setTasks] = useState<CommandTask[] | undefined>(undefined);
   const [sourcesModal, setSourcesModal] = useState<SourceInfo>();
   const pageRef = useRef<HTMLElement>(null);
-  const { contextHolder: modalJupyterLabContextHolder, modalOpen: openJupyterLabModal } =
```

Comment:
When a user is on the Task list within a workspace the dropdown should be required and default to the current workspace. It looks like due to removing the `workspace` prop in `useModalJupyterLab` this is no longer working correctly. 

![2A2D6B35-624C-4169-982A-4F3B6D2E65F1](https://user-images.githubusercontent.com/103522725/218822686-5525a4f1-98a5-4adf-ad1b-2d3b0a6eedd0.jpeg)

Reply:
ah thanks for catching that, updated